jp/components/expansion_panel.md

```diff
@@ -222,32 +163,151 @@ NgModule({
+<!-- in expansion-component.component.html -->
+    <igx-expansion-panel>
+    ...
+        <igx-expansion-panel-header [iconPosition]="'right'">
+        </igx-expansion-panel-header>
+    </igx-expansion-panel>
+```
+> [`iconPosition`]({environment:angularApiUrl}/classes/igxexpansionpanelheadercomponent.html#iconposition) プロパティは、`RTL` で使用できます。たとえば、**right** に表示するよう設定したアイコンは、RTL 設定時にヘッダーの左端に表示されます。
+
+コンテンツを渡してコントロールで使用するデフォルト アイコンは、`igx-expansion-panel-icon` タグでをオーバーライドできます。
+```html
+<!-- in expansion-component.component.html -->
+    <igx-expansion-panel>
+    ...
+        <igx-expansion-panel-header [iconPosition]="'right'">
+        ...
+            <igx-expansion-panel-icon>
+                <span class="example-icon" *ngIf="panel.collapsed">Show more</span>
+                <span class="example-icon" *ngIf="!panel.collapsed">Collapse</span>
+            </igx-expansion-panel-icon>
+        </igx-expansion-panel-header>
+    </igx-expansion-panel>
+```
+```css
+<!-- in expansion-component.component.scss -->
+.example-icon {
+    font-size: 12px;
+    font-weight: 600px;
+}
+```
+このコンポーネントは、パネルの縮小時と完全に展開した後に「更に表示」を描画します。
+
+### テーマのインポート
+[Ignite UI for Angular テーマ](themes/index.md)を使用して、**igx-expansion-panel** の外観を変更できます。次に、より安定する方法でカスタマイズします。
+
+はじめに、テーマ エンジンによって公開されている関数を使用するために、スタイル ファイルに `index` ファイルをインポートする必要があります。 
+
+```scss
+// in styles.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+```
+
+### パレットと色の定義
+`index` ファイルをインポート後、[`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) 関数を使用して、カスタムテーマで使用するいくつかの色変数を定義できます。コンポーネントで使用するメイン カラーを指定できる独自のカラー[パレット](themes/palette.md)と組み合わせて、`igx-expansion-panel` スタイルにこれらを使用します。
+はじめにカスタム パレットを定義し、メイン カラーを渡します。
+```scss
+// in styles.scss
+$my-primary-color:#72da67;
+$my-secondary-color: #7344df81;
+$my-info-color:rgba(0, 0, 0, 0.54)
+
+$my-color-palette: igx-palette(
+    $primary: $my-primary-color,
+    $secondary: $my-secondary-color,
+    $info: $my-info-color
+);
+```
+
+カスタム パレットと適用された色を確認するには、これらをテーマ関数に渡す必要があります。
+次に[`カスタム テーマを作成`](themes/component-themes.md#creating-themes)し、定義済みの `igx-expansion-panel-theme` パラメーターに色を渡します。これらの特定のパラメーターを変更してコンポーネントを希望する外観にできると仮定します。以前に作成した[パレット](themes/palette.md)を使用して、定義した色に基づいて新しい色を作成すると非常に便利です。
+```scss
+// in styles.scss
+$custom-panel-theme: igx-expansion-panel-theme(
+    $palette: $my-color-palette,
+    $header-background: igx-color($my-color-palette, "primary", 200),
+    $header-focus-background: igx-color($my-color-palette, "primary", 300),
+    $header-title-color: igx-color($my-color-palette, "info"),
+    $header-icon-color: igx-color($my-color-palette, "info"),
+    $body-background: igx-color($my-color-palette, "secondary", 100),
+    $border-radius: 0.35em
+);
+```
+
+### 適用
+あとは新しく作成したテーマを適切にスコープするだけです。
+
+#### グローバル
+この新しく作成された `igx-expansion-panel テーマ`をアプリにグローバルに適用する場合、必要なのは、アプリのルート スタイル ファイルにテーマを含め、[igx-expansion-panel mixin]({environment:sassApiUrl}/index.html#mixin-igx-expansion-panel) に渡すだけです。
```

Comment:
It appears that the `globally` link - (themes/component-themes.md#creating-themes) - is missing.